user-manual/ingest/manual-normalization.rst

```diff
@@ -4,261 +4,357 @@
+
+#. When you reach the normalization microservice on the Ingest tab, choose
+   **Normalize manually** from the dropdown menu.
+
+#. Manually normalize your files.
+
+#. Using an SFTP file browser or SSH in your command line, go to your
+   pipeline's Processing Location and find the approveNormalization watched
+   directory. The default location is
+   ``/var/archivematica/sharedDirectory/watchedDirectories/approveNormalization/preservation/``,
+   which will contain your transfer directory. You may need to look at your
+   :ref:`Storage Service <storageservice:index>` configuration to confirm the
+   Processing Location defined for your pipeline.
+
+#. Copy your manually normalized derivatives into your transfer directory. You
+   will need to create a directory structure containing a
+   ``manualNormalization`` directory. Inside of the ``manualNormalization``
+   directory, create directories called ``access`` and ``preservation`` (you do
+   not need both - if you only have preservation copies, create only the
+   ``preservation`` directory, and vice versa). Place your manually normalized
+   files in the appropriate directory.
+
+   .. code:: bash
+
+      my-manually-normalized-material/
+      ├── manualNormalization
+      │   └── access
+      │       └── pepperbox-poppy.gif
+      │   └── preservation
+      │       ├── hooded-pitcher-plant.tif
+      │       └── pepperbox-poppy.tif
+      ├── hooded-pitcher-plant.png
+      ├── ornamental-onion.png
+      └── pepperbox-poppy.png
 
-   If you think you will need to use this aspect of the feature, please note
-   that Archivematica does not offer the option to add PREMIS eventDetail by
-   default. Instead, you must use the preconfigured choices in the processing
-   configuration section of the Administrative tab of the dashboard to adjust
-   your settings prior to ingest. Deselect the Reminder in the Processing
-   Configuration settings.
+   In the above example, preservation and access derivatives were created for
+   ``pepperbox-poppy.png``, while only a preservation copy was created for
+   ``hooded-pitcher-plant.png``.
 
-.. image:: images/ReminderInProcessingConfig.*
-   :align: center
-   :width: 60%
-   :alt: Deselecting Reminder in Processing Configuration settings.
+   Note that the filenames must match in order for Archivematica to recognize
+   the links between the original file and its derivatives.
 
+#. Return to the dashboard and approve normalization.
 
-If you include manually normalized files in the initial transfer (option 1) or
-during processing (option 2), the procedure for adding PREMIS eventDetail is
-the same.
+#. Once the AIP is created, you will find that the originals and their
+   preservation derivatives are now located in the ``data`` directory of the AIP
+   as usual. Note that the ``manualNormalization`` directory and subfolders were
+   deleted once normalization was complete. If you had access copies, a DIP will
+   be created as well.
```

Comment:
This has been the best description of this workflow I have ever read! :smile_cat:

Reply:
Impressive, since I've never actually done it.